README.md

```diff
@@ -61,6 +55,33 @@ if (import.meta.url === pathToFileURL(process.argv[1]).href) {
 }
 ```
 
+### For Common JS
+
+```js
+const url = require("url");
+
+module.exports.handle = (event, context, callback) => {
+  return {
+    statusCode: 201,
+    body: JSON.stringify({
+      message: "Hello World!",
+    }),
+    headers: {
+      "Content-Type": "application/json",
+    },
+  };
+};
+
+// This will execute when testing locally, but not when the function is launched
+if ("file://" + __filename === url.pathToFileURL(process.argv[1]).href) {
```

Comment:
I did this to have the same behaviour for CJS and MJS files, before CJS files would require to have a specific en variable to be set.